packages/core/src/components/Form.tsx

```diff
@@ -40,6 +40,7 @@ import _pick from 'lodash/pick';
 import _toPath from 'lodash/toPath';
 
 import getDefaultRegistry from '../getDefaultRegistry';
+import { forEach } from 'lodash';
```

Comment:
Oh shoot, sorry I missed this. Can you update the imports to explicitly import `forEach` like the ones above, putting it before `_get`